state/subnets.go

```diff
@@ -174,24 +176,6 @@ func (s *Subnet) ProviderNetworkId() network.Id {
 	return network.Id(s.doc.ProviderNetworkId)
 }
 
-// Validate validates the subnet, checking the CIDR, and VLANTag, if present.
-func (s *Subnet) Validate() error {
```

Comment:
After reworking AddSubnet(), this validation fits better on SubnetInfo{}, moved.